docs/streams/developer-guide/config-streams.html

```diff
@@ -300,12 +306,12 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
               set by the user or all serdes must be passed in explicitly (see also default.key.serde).</td>
             <td><code class="docutils literal"><span class="pre">null</span></code></td>
           </tr>
-          <tr class="row-even"><td>default.windowed.key.serde.inner</td>
+          <tr class="row-even"><td>default.windowed.key.serde.inner (Deprecated. Use windowed.inner.class.serde instead.)</td>
             <td>Medium</td>
             <td colspan="2">Default serializer/deserializer for the inner class of windowed keys, implementing the <code class="docutils literal"><span class="pre">Serde</span></code> interface.</td>
             <td><code class="docutils literal"><span class="pre">null</span></code></td>
           </tr>
-          <tr class="row-odd"><td>default.windowed.value.serde.inner</td>
+          <tr class="row-odd"><td>default.windowed.value.serde.inner (Deprecated. Use windowed.inner.class.serde instead.)</td>
             <td>Medium</td>
             <td colspan="2">Default serializer/deserializer for the inner class of windowed values, implementing the <code class="docutils literal"><span class="pre">Serde</span></code> interface.</td>
             <td><code class="docutils literal"><span class="pre">null</span></code></td>
```

Comment:
I just see by change, that we use a different formatting for marking a deprecated config:
```
<tr class="row-even"><td>default.dsl.store</td>
  <td>Low</td>
  <td colspan="2">
    DEPRECATED] The default state store type used by DSL operators. Deprecated in
    favor of <code>dsl.store.suppliers.class</code>
    </td>
```

I think we should use a unified formatting -- don't have a preference which one.